apps/test-examples/App.js

```diff
@@ -12,6 +12,7 @@ import Test1790 from './src/Test1790';
 import Test1813 from './src/Test1813';
 import Test1845 from './src/Test1845';
 import Test1986 from './src/Test1986';
+import Test2054 from './src/Test2054';
```

Comment:
Please, change the test number to this PR or corresponding issue

Reply:
Test number and order of imports have been updated.